app/backend/src/tests/test_events.py

```diff
@@ -1,15 +1,14 @@
 from datetime import timedelta
 
-from sqlalchemy.sql.expression import update
-from couchers.models import EventOccurrence
-
 import grpc
 import pytest
 from google.protobuf import wrappers_pb2
 from psycopg2.extras import DateTimeTZRange
+from sqlalchemy.sql.expression import update
 
 from couchers import errors
 from couchers.db import session_scope
+from couchers.models import EventOccurrence
 from couchers.utils import Timestamp_from_datetime, now, to_aware_datetime
```

Comment:
Are these header rearrangements caused by a new version of `isort`? We should be doing them all at once in a separate single PR then.

Reply:
Since it's not pinned to a version in `requirements.txt`, I guess it's possible that I installed a newer version than the one you/Aapeli have on your machine as it's been the first time I really touched the backend. Maybe we should pin that to a version too? 🤷

Reply:
It might be a PR that was merged which we did not run `isort` on. I've opened https://github.com/Couchers-org/couchers/pull/2030 to add a check to CI.

Fine by me to do this change in this PR.